src/user/test/all-get-user.controller.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { HttpStatus, ValidationPipe } from '@nestjs/common';
```

Comment:
maybe better change name of the file to `get-all-user.controller...`?

Reply:
In this case it looks weird, but I'm using the convention of endpoint-httpmethod-module.controller...